Src/BootCamp.Chapter/BMI.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class BMI
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename class 'BMI' to match pascal case naming rules, consider using 'Bmi'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4852921)